src/Expanders.jl

```diff
@@ -411,6 +411,7 @@ end
 
 function Selectors.runner(::Type{EvalBlocks}, x, page, doc)
     sandbox = Module(:EvalBlockSandbox)
+    eval(sandbox, :(import Compat.Markdown))
```

Comment:
Why do we need this?

Reply:
Not sure, but it silenced a bunch of warnings. I think because previously `Markdown` was always available by default in all modules since they all have implicit `using Base` in them.

Reply:
```
WARNING: importing deprecated binding Base.Markdown into EvalBlockSandbox.
WARNING: Base.Markdown is deprecated, run `using Markdown` instead
  likely near /home/fredrik/.julia/v0.7/Documenter/test/examples/make.jl:91
```

Reply:
I guess we `eval` things into the module which references `Markdown`.

Reply:
Ah, good catch, should be fixed now.